actionpack/lib/action_dispatch/middleware/params_parser.rb

```diff
@@ -1,9 +1,15 @@
-require 'active_support/core_ext/hash/conversions'
 require 'action_dispatch/http/request'
-require 'active_support/core_ext/hash/indifferent_access'
```

Comment:
Are these no longer used?

I can't see any explanation for why you're making these changes. (Maybe because @seuros asked you to squash it away, before anyone had a chance to look?)